src/main/target/KAKUTEF4/target.c

```diff
@@ -29,7 +29,7 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM3, CH4, PB1, TIM_USE_MOTOR, TIMER_OUTPUT_ENABLED, 0),  // S2_OUT - DMA1_ST2
     DEF_TIM(TIM2, CH4, PA3, TIM_USE_MOTOR, TIMER_OUTPUT_ENABLED, 1),  // S3_OUT - DMA1_ST6
     DEF_TIM(TIM2, CH3, PA2, TIM_USE_MOTOR, TIMER_OUTPUT_ENABLED, 0),  // S4_OUT - DMA1_ST1
-    DEF_TIM(TIM5, CH2, PA1, TIM_USE_MOTOR, TIMER_OUTPUT_ENABLED, 0),  // S5_OUT - DMA1_ST4
+    DEF_TIM(TIM5, CH1, PA0, TIM_USE_MOTOR, TIMER_OUTPUT_ENABLED, 0),  // S5_OUT - DMA1_ST2
```

Comment:
I assume that you are aware of, and ok with the fact that this changes the default pin assignment, and will require manual reconfiguration for existing users?